crown-court-proceeding/src/test/java/uk/gov/justice/laa/crime/crowncourt/integration/WiremockIntegrationTest.java

```diff
@@ -1,41 +1,34 @@
 package uk.gov.justice.laa.crime.crowncourt.integration;
 
+import static com.github.tomakehurst.wiremock.client.WireMock.post;
+import static org.springframework.boot.test.context.SpringBootTest.WebEnvironment.DEFINED_PORT;
+
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.github.tomakehurst.wiremock.WireMockServer;
 import com.github.tomakehurst.wiremock.client.WireMock;
-import org.junit.jupiter.api.AfterEach;
+import java.util.Map;
+import java.util.UUID;
 import org.junit.jupiter.api.TestInstance;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.autoconfigure.actuate.observability.AutoConfigureObservability;
 import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.cloud.contract.wiremock.AutoConfigureWireMock;
 import org.springframework.context.annotation.Import;
 import org.springframework.http.MediaType;
 import org.springframework.test.annotation.DirtiesContext;
+import org.wiremock.spring.EnableWireMock;
+import org.wiremock.spring.InjectWireMock;
 import uk.gov.justice.laa.crime.crowncourt.config.IntegrationTestConfiguration;
 
-import java.util.Map;
-import java.util.UUID;
-
-import static com.github.tomakehurst.wiremock.client.WireMock.post;
-import static org.springframework.boot.test.context.SpringBootTest.WebEnvironment.DEFINED_PORT;
-
+@EnableWireMock
 @DirtiesContext
 @AutoConfigureObservability
-@AutoConfigureWireMock(port = 9998)
 @Import(IntegrationTestConfiguration.class)
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
 @SpringBootTest(classes = IntegrationTestConfiguration.class, webEnvironment = DEFINED_PORT)
 public abstract class WiremockIntegrationTest {
 
-    @Autowired
-    protected WireMockServer wiremock;
-
-    @AfterEach
-    void clean() {
-        wiremock.resetAll();
-    }
+    @InjectWireMock
+    protected static WireMockServer wiremock;
```

Comment:
Do we need to inject the Wiremock server here instead of just calling the static method to create the stubs? Just wondering what is the reason for the change?

Reply:
Oh cool, nice to avoid the unnecessary depepdencies and vulnerabilities. Thanks for the explanantion.